src/navigations/types.tsx

```diff
@@ -0,0 +1,17 @@
+import { RouteProp } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+
+export type RootStackParamList = {
+  Home: undefined;
+  AddTask: undefined;
```

Comment:
```suggestion
  AddTask: React.ReactNode; 
```

undefined 대신 이거로 바꾸면 될걸 아마

Reply:
React.FC 였네요 :)